kythe-verification/testdata/basic/PatSyn.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE PatternSynonyms #-}
```

Comment:
Why do we have the same file twice?

Reply:
This tests the Kythe output (the others the intermediate, still-Haskell layer).